ui/component/claimRepostAuthor/view.jsx

```diff
@@ -47,10 +47,14 @@ function ClaimRepostAuthor(props: Props) {
 
   return (
     <div className="claim-preview__repost-author">
-      <Icon icon={ICONS.REPOST} size={10} className="claim-preview__repost-icon" />
-      <I18nMessage tokens={{ repost_channel_link: <UriIndicator link uri={repostChannelUrl} /> }}>
-        %repost_channel_link% reposted
-      </I18nMessage>
+      <div className="claim-preview__repost-ribbon">
+        <Icon icon={ICONS.REPOST} size={10} className="claim-preview__repost-icon" />
+        repost
```

Comment:
Localization for `repost` is lost without `__()`.
I have a feeling not all languages will fit the slanted area. e.g. `ਦੁਬਾਰਾ ਪੋਸਟ ਕਰੋ

Reply:
I've replaced it with `__('Repost')`, but changing the language of the interface doesn't really switch any labels for me. Neither in my version nor on Odysee.com. Punjabi isn't even in my list. How am I supposed to test that? O_o

Reply:
I've replaced it with `__('Repost')`, but it doesn't look like there's a Punjabi translation for that particular label. I've tested it in English, German & Japanese. Looks fine so far, but I guess there are still many language related issues in regards to space. However, the current version of Odysee has exactly the same issue. It's just something to keep in mind moving forward.

Reply:
Jeremy Kauffman suggested to get completely rid of the label in the list view since other sites like Twitter and Tumblr only use icons to indicate a repost. I've decided to remove it from the tiles as well to keep things consistent and also to solve this issue. The icon now looks a little bit out of place without the text, but at least the overflow is no longer an issue. I'm probably gonna redesign this ribbon once version 1.0 is out.

Reply:
Looks good, actually.  I like it.